substrate/frame/staking-async/ah-client/src/mock.rs

```diff
@@ -0,0 +1,120 @@
+// This file is part of Substrate.
+
+// Copyright (C) Parity Technologies (UK) Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// 	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! Mock runtime for pallet-staking-async-ah-client tests.
+
+#![cfg(any(test, feature = "runtime-benchmarks"))]
+
+use crate::*;
+use frame_support::{derive_impl, parameter_types, weights::Weight};
+use pallet_staking_async_rc_client as rc_client;
+use sp_runtime::{BuildStorage, Perbill};
+use sp_staking::offence::{OffenceSeverity, OnOffenceHandler};
+
+type Block = frame_system::mocking::MockBlock<Test>;
+
+frame_support::construct_runtime!(
+	pub enum Test
+	{
+		System: frame_system,
+		StakingAsyncAhClient: crate,
+	}
+);
+
+#[derive_impl(frame_system::config_preludes::TestDefaultConfig)]
+impl frame_system::Config for Test {
+	type Block = Block;
+	type AccountData = ();
+}
+
+pub struct MockSendToAssetHub;
+impl SendToAssetHub for MockSendToAssetHub {
```

Comment:
Seems like `SendToAssetHub` is implemented for `()`, but it panics and is not useful here.